tools/coverity/misra.config

```diff
@@ -10,26 +10,44 @@
             category: "Advisory",
             reason: "Allow inclusion of function like macros."
         },
+        {
+            deviation: "Rule 2.5",
+            reason: "Allow unused macros. Library headers may define macros intended for the application's use, but not used by a specific file."
+        },
+        {
+            deviation: "Rule 3.1",
+            category: "Required",
+            reason: "Allow nested comments. Documentation blocks contain comments for example code."
```

Comment:
Can you point some instances of nested comments?
Is the deviation actually because of `//` in hyperlinks within comments?

Reply:
+1. I think // is used in hyperlinks such as the ones used in the license headers in all the files